sdks/csharp.md

```diff
@@ -2,7 +2,11 @@
 
 ### Download & Install
 
-Coming Soon
+```
+nuget install Bandwidth.Sdk -OutputDirectory packages -Version 1.0.0-beta
+```
+
```

Comment:
Add a disclaimer that this is the beta and there _could_ be updates with official release.  Then we can take 'er live.